### DISCUSSION TITLE
Suggestions for Ethereum projects.

### DISCUSSION THREAD

**Discussion by oneworldcoder** [EddieHubCommunity/support#338] [Q&A]
Hello everyone! 

I am a beginner to Ethereum programming (in Solidity). I am looking for interesting projects to contribute to. Can you guys recommend me some project links? 

Have a great day.
Thank you.

**Comment by eddiejaoude** [comment]
Hi @oneworldcoder , I am not familiar with Ethereum, but hopefully someone else in the community can share some light on this 🤓
